<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_right{
	background-color: rgb(255, 255, 255);
}
QLabel#label_left{
	
	background-color: rgb(0, 170, 255);
}
QLabel#label_up{
	
	background-color: rgb(235, 235, 235);
}</string>
  </property>
  <widget class="QLabel" name="label_up">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>2</y>
     <width>830</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>830</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>830</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mini">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_up">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>102</y>
     <width>832</width>
     <height>422</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_left">
      <property name="minimumSize">
       <size>
        <width>415</width>
        <height>420</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>415</width>
        <height>420</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_right" native="true">
      <property name="minimumSize">
       <size>
        <width>415</width>
        <height>420</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>415</width>
        <height>420</height>
       </size>
      </property>
      <widget class="QLabel" name="label_account_passwd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>用户名密码登录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_handof_captche">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>验证码登录</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>103</y>
         <width>362</width>
         <height>292</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_user_name">
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_passwd">
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_remember">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>记住我</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_forget_passwd">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>忘记密码？</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_register">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>没有账号？立即注册！</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_login">
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_other_login">
          <property name="font">
           <font>
            <family>楷体</family>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>其他方式登录</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_QQ">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_wx">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_wb">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
